builder/digitalocean/builder.go

```diff
@@ -48,6 +50,8 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 	client := godo.NewClient(oauth2.NewClient(context.TODO(), &apiTokenSource{
 		AccessToken: b.config.APIToken,
 	}))
+	client.UserAgent = useragent.String(version.PluginVersion.FormattedVersion())
```

Comment:
I can see the need for having a more specific UserAgent over the default [godo UserAgent](https://github.com/digitalocean/godo/blob/v1.78.0/godo.go#L25). Overall the change looks good to me. But the same change is needed for the post-processor. 

Can you please update the UserAgent for the client used in the [post-processor](https://github.com/hashicorp/packer-plugin-digitalocean/blob/c10f794e4005eec0cd9628ada13524b47b2a22c8/post-processor/digitalocean-import/post-processor.go#L202), as well.